core/src/main/scala/org/apache/spark/storage/memory/MemoryStore.scala

```diff
@@ -86,7 +91,8 @@ private[spark] class MemoryStore(
 
   // A mapping from taskAttemptId to amount of memory used for unrolling a block (in bytes)
   // All accesses of this map are assumed to have manually synchronized on `memoryManager`
-  private val unrollMemoryMap = mutable.HashMap[Long, Long]()
+  private val onHeapUnrollMemoryMap = mutable.HashMap[Long, Long]()
+  private val offHeapUnrollMemoryMap = mutable.HashMap[Long, Long]()
```

Comment:
can you add a comment here to say the off heap one is only used for bytes? (Is that true)